src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -48,9 +48,9 @@ public PersonCard(Person person, int displayedIndex) {
         this.person = person;
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
-        phone.setText(person.getPhone().value);
-        address.setText(person.getAddress().value);
-        email.setText(person.getEmail().value);
+        phone.setText("Phone: " + person.getPhone().value);
+        address.setText("Address: " + person.getAddress().value);
+        email.setText("Email: " + person.getEmail().value);
         visitList.setText(person.getVisitList().toString());
```

Comment:
This is still gonna be `[]` for now?

Reply:
which one?

Reply:
Was referring to `visitList.setText`, but I just saw the screenshot. Looks great and thanks for including it!